test/stable/common/configmap.rb

```diff
@@ -0,0 +1,264 @@
+# frozen_string_literal: true
```

Comment:
This test won't run with the current file name. Test file names need to end with `_spec.rb` for them to be detected.